apps/judicial-system/backend/src/app/modules/event-log/eventLog.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Body, Controller, Inject, Post } from '@nestjs/common'
+import { ApiCreatedResponse } from '@nestjs/swagger'
+
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+
+import { EventType, User } from '@island.is/judicial-system/types'
+
+import { EventLogService } from './eventLog.service'
+
+@Controller('api/event-log')
+export class EventLogController {
+  constructor(
+    private readonly eventLogService: EventLogService,
+    @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
+  ) {}
+
+  @Post('log-login')
+  @ApiCreatedResponse({ description: 'Logs user login in event log' })
+  logLogin(@Body() user: User): Promise<void> {
```

Comment:
Please consider creating a more general log method that receives event log info like national id and type, that way we could easily reuse the code for future event logs